src/util/recorder.ts

```diff
@@ -102,20 +102,24 @@ export class Recorder {
     this.collDir = collDir;
 
     this.archivesDir = path.join(this.collDir, "archive");
-    this.tempdir = path.join(this.collDir, "tmp-dl");
+    this.tempdir = path.join(os.tmpdir(), "tmp-dl");
     this.tempCdxDir = path.join(this.collDir, "tmp-cdx");
 
     fs.mkdirSync(this.tempdir, { recursive: true });
     fs.mkdirSync(this.archivesDir, { recursive: true });
     fs.mkdirSync(this.tempCdxDir, { recursive: true });
 
+    const prefix = crawler.params.warcPrefix || "rec";
     const crawlId = process.env.CRAWL_ID || os.hostname();
-    const filename = `rec-${crawlId}-${timestampNow()}-${this.workerid}.warc`;
+    const filenameTemplate = `${prefix}-${crawlId}-$ts-${this.workerid}.warc${
```

Comment:
Do you mean the timestamp? Previously, the WARCWriter would only write to a single WARC with filename passed in, now it can open multiple files, so this becomes a filename template, and the only change is the timestamp for each file that is opened.

Reply:
I suppose this could be added as a comment to make this clearer!

Reply:
Ah right! I think the filename makes it clear enough in retrospect! I just missed that change in the diff